std/meta.d

```diff
@@ -355,6 +376,62 @@ unittest
 
     alias TL = NoDuplicates!(Types);
     static assert(is(TL == AliasSeq!(int, long, float)));
+
+    alias LongList = AliasSeq!(int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int,
+        int,int,int,int,int,int,int,int,int,int);
```

Comment:
This can be replaced with a much more concise version by using `std.meta.Repeat`.